Made changes in JavaScript:data-types.md (#1172)
Codecademy/docs: content/javascript/concepts/data-types/data-types.md

  - 'paths/front-end-engineer-career-path'
---

**Data types** are how data in programming is classified. In JavaScript, there are eight fundamental data types:

> ## Primitive Data Types

Reviewer: Make sure to add spacing between paragraphs and headings!

Reviewer: Check the rest of the entry to maintain this standard

> The below 7 in the table are considered primitive data types. They are the most basic data types in the language. Objects are a little more complex.

Reviewer: ```suggestion
The table below holds the seven primitive data types. Objects are a little more complex.
```

| Data Type | Wrapper Object   | Description                                                                                                                                   |
|-----| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------- |

> | `number` | [`Number`](../number-methods/number-methods.md)    | Any number, including numbers with decimals: `1`, `-2`, `99`, `3.14`.                                                                         |

Reviewer: Make sure to link out properly, the urls should look like https://www.codecademy.com/resources/docs/...

| `bigInt` | `BigInt`    | Any number, greater than 2^53-1 or less than -(2^53-1) with `n` appended to the number: `1234567890123456n`.                                  |

> | `string` | [`String`](../strings/strings.md)    | Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single quotes `''` or double quotes `""`. |

Reviewer: ```suggestion
| `string` | [`String`](../strings/strings.md)    | Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single `''` or double quotes `""`. |
```

| `boolean` | `Boolean`   | This data type only has two possible values — either `true` or `false`.                                                                       |
| `null` | `Null`      | This data type represents the intentional absence of a value, and is represented by the keyword `null`.                                       |
| `undefined` | `Undefined` | This data type is denoted by the keyword `undefined`. It also represents the absence of a value though it has a different use than `null`.    |
| `symbol` | `Symbol`    | A newer feature to the language, symbols are unique identifiers, useful in more complex coding. No need to worry about these for now.         |

## Objects

> Collection of related Data.<br/><br />                                                                                                                  

Reviewer: Wondering what the break is for — we can add a spacing, manually.

| `bigInt` | `BigInt`    | Any number, greater than 2^53-1 or less than -(2^53-1) with `n` appended to the number: `1234567890123456n`.                                  |
| `string` | [`String`](https://www.codecademy.com/resources/docs/javascript/strings)    | Any grouping of characters on your keyboard (letters, numbers, spaces, symbols, etc.) surrounded by single `''` or double `""`. |
| `boolean` | `Boolean`   | This data type only has two possible values — either `true` or `false`.                                                                       |
| `null` | `Null`      | This data type represents the intentional absence of a value, and is represented by the keyword `null`.                                       |
| `undefined` | `Undefined` | This data type is denoted by the keyword `undefined`. It also represents the absence of a value though it has a different use than `null`.    |
| `symbol` | `Symbol`    | A newer feature to the language, symbols are unique identifiers, useful in more complex coding. No need to worry about these for now.         |

> 
> ## Objects
> Collection of related Data.    

Reviewer 2: ```suggestion
## Objects

[Objects](https://www.codecademy.com/resources/docs/javascript/objects) are a little more complex since they are collections of related data. 
```

> 
> 
> At first, eight types may not seem like that many, but they are the foundation that helps programmers build a whole new world.
> 

Reviewer 2: ```suggestion

```

I would omit this sentence as I don't think it adds to the manual-purposes of a Docs entry.

  - 'paths/front-end-engineer-career-path'
---

> **Data types** are how data in programming is classified. In JavaScript, there are eight fundamental data types: 

Reviewer 2: Let's also make a similar change to the meta-`Description` on line # 3:

> Description: 'Data types are used to classify eight fundamental values used for programming in JavaScript, including primitive and object types.'
